machine.go

```diff
@@ -456,7 +460,10 @@ func (m *Machine) Start(ctx context.Context) error {
 // Shutdown requests a clean shutdown of the VM by sending CtrlAltDelete on the virtual keyboard
 func (m *Machine) Shutdown(ctx context.Context) error {
```

Comment:
Doesn't Firecracker return an error? Does the SDK need to know that?

Reply:
I still want to remove this check if possible. While we know Firecracker cannot send Ctrl + Alt + Del on aarch64 microVMs, it would be better to let Firecracker make a call, rather than doing that in SDK.

Reply:
ok, let me create an Issue for this check in Firecracker repository.